src/parser/jobs/drg/modules/index.js

```diff
@@ -6,6 +6,9 @@ import Debuffs from './Debuffs'
 import Procs from './Procs'
 //import RotationWatchdog from './RotationWatchdog'
 import Weaving from './Weaving'
+import Jumps from './Jumps'
+import Positionals from './Positionals'
```

Comment:
These should also be kept alphabetical, both in the import list and in the export array.